R/bambu.R

```diff
@@ -115,28 +115,27 @@
 #' se <- bambu(reads = test.bam, annotations = gr, 
 #'     genome = fa.file,  discovery = TRUE, quant = TRUE)
 #' @export
-bambu <- function(reads = NULL, rcFile = NULL, rcOutDir = NULL,
+bambu <- function(reads, rcOutDir = NULL,
     annotations = NULL, genome = NULL, stranded = FALSE, ncore = 1, NDR = 0.1,
     yieldSize = NULL, opt.discovery = NULL, opt.em = NULL, trackReads = FALSE, 
     returnDistTable = FALSE, discovery = TRUE, quant = TRUE, fusionMode = FALSE, 
     verbose = FALSE, lowMemory = FALSE) {
     if (!(discovery+quant)) stop("At least 1 of discovery and quant must be 
     TRUE. Rerun with either 1 or both parameters as TRUE")
     if(is.null(annotations)) { annotations = GRangesList()
-    } else annotations <- checkInputs(annotations, reads, readClass.file = rcFile,
+    } else annotations <- checkInputs(annotations, reads,
             readClass.outputDir = rcOutDir, genomeSequence = genome)
-    if(!is.null(reads)) genomeSequence <- checkInputSequence(genome)
+    genomeSequence <- checkInputSequence(genome)
```

Comment:
with the read class files, is the genome still required? Maybe not right? In which case we might have to catch this here

Reply:
Resolved 330e063d9f688f37044ec8b29f396e2007636113
Now it is checked in checkInputs() that users arn't using bam with no genome